src/index.html

```diff
@@ -15,8 +15,82 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="me"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo__link"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <a
+          href="#apple"
+          class="nav__link is-active"
+        >
+          <p>apple</p>
+        </a>
+        <a
+          href="#samsung"
+          class="nav__link"
+        >
+          samsung
+        </a>
+        <a
+          href="#smartphones"
+          class="nav__link"
+        >
+          smartphones
+        </a>
+        <a
+          href="#laptop&computers"
+          class="nav__link"
+          data-qa="hover"
+        >
+          laptops & computers
+        </a>
+        <a
+          href="#gadgets"
+          class="nav__link"
+        >
+          gadgets
+        </a>
+        <a
+          href="#tablets"
+          class="nav__link"
+        >
+          tablets
+        </a>
+        <a
+          href="#photo"
+          class="nav__link"
+        >
```

Comment:
This looks like list of elements. For this purpose there are semantic tags as `<ul>` or `<ol>` and `<li>` tags. This is also requrement for this task.